<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>review_window</class>
 <widget class="QMainWindow" name="review_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lblSelect">
        <property name="text">
         <string>Select a market to review</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbMarketSelect"/>
      </item>
      <item>
       <widget class="QLabel" name="lblScore">
        <property name="text">
         <string>Assign a score</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbScores"/>
      </item>
      <item>
       <widget class="QLabel" name="lblText">
        <property name="text">
         <string>Share your experience here (optional)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="tReviewBody"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QPushButton" name="btnReview">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
